editstyle.css

```diff
@@ -262,9 +264,139 @@ nav ul li a:hover {
   font-size: 16px;
   cursor: pointer;
   margin-top: 12px;
-} 
+}
 .close-btn:hover {
   background-color: #d32f2f;
 }
 
-  
\ No newline at end of file
+/* Responsive Styles */
+
```

Comment:
Use Media queries from Bootstrap or tailwind for better Responsiveness breakpoints

Reply:
ok